Clients/src/presentation/components/PluginWidgets/utils/iconMapping.ts

```
@@ -0,0 +1,661 @@
+  CloudSun,
+  CloudRain,
+  Snowflake,
+  Wind,
+  Leaf,
+  Trees,
+  Flower,
+
+  // Misc
+  Globe,
+  Map,
+  MapPin,
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)